msautotest/pytest.ini

```diff
@@ -2,3 +2,5 @@
 python_files = *.py
 testpaths = api misc gdal query renderers wxs sld mspython
 norecursedirs = php mssql
+env =
+    MAPSERVER_CONFIG_FILE=../etc/mapserv.conf
```

Comment:
This file doesn't exist? Should it be `mapserver-sample.conf `?

Reply:
@rouault mentioned this configuration was not exactly intuitive but the file referenced it's actually msautotest/etc/mapserv.conf. That config file is just the bare minimum and only suitable for use with msautotest. The etc/mapserver-sample.conf is different. I included that as a kitchen sink base file for users to start from with all the options and heavily commented. Kinda like you see in an Apache configuration file. Perhaps that sample file could be stored someplace else like the new share directory added with the ogcapi work.

--Steve

Reply:
Oops..! It is right above this comment in the GitHub web UI. Thanks for the clarification.